pkg/controllers/disruption/multinodeconsolidation.go

```diff
@@ -57,15 +58,20 @@ func (m *MultiNodeConsolidation) ComputeCommand(ctx context.Context, disruptionB
 	disruptableCandidates := make([]*Candidate, 0, len(candidates))
 	constrainedByBudgets := false
 	for _, candidate := range candidates {
+		// Filter out empty candidates 
+		if candidate.NodeClaim.StatusConditions().GetCondition(v1beta1.Empty) != nil {
```

Comment:
I need to clear my modcache and add this commit: https://github.com/Bryce-Soghigian/karpenter-core/commit/49ce67fb94eb7a1c3e22ad706ffe093ab845d53d I believe.